### PR TITLE
feat(webapp): move delegate names below graph

### LINKED ISSUE #102: Test placing delegate names below graph
image

### LINKED ISSUE #103: No indicator for x axis
Add date in x axis for per election and per delegate views

We need a label for the x asis
image

image

### LINKED ISSUE #114: Change Exchange rate Label
The Label for `eos_exchange` rate should read `Exchange Rate:`

image

### LINKED ISSUE #115: Change table header labels
Name should be Term (periodo en español)

The table header should replace word `earned` for `distributed` 

image

### LINKED ISSUE #117: Add thousands separator for table
Add a thousands separator to table numbers

image

### LINKED ISSUE #120: Switch exchange rate
create a switch button that shows or hides exchange rates

image

image

exchange rate is displayed by default but then users can choose to remove it

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #114 
- Resolve #102
- Resolve #103
- Resolve #120 
- Resolve #117 
- Resolve #115

### Steps to test

1. Run the project
2. Open in your browser localhost:3000
3. Click on income in the sidebar
4. See the new changes

- Now
image

- Before
image

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: webapp/src/hooks/customHooks/useIncomeReportState.js [hunk 1]
```diff
@@ -53,29 +53,37 @@ const useIncomeReportState = () => {
...
-    const data = await response
-
-    setCurrencyBalance(data[0] || '')
+    try {
+      const response = eosApi.getCurrencyBalance(
+        'eosio.token',
+        'genesis.eden',
+        'EOS'
+      )
+      const data = await response
```
Comment (leisterfrancisco) [inline webapp/src/hooks/customHooks/useIncomeReportState.js:62]: why two lines for that? what if you only put `await eosApi.`

File: webapp/src/hooks/customHooks/useIncomeReportState.js [hunk 2]
```diff
@@ -53,29 +53,37 @@ const useIncomeReportState = () => {
...
-      'genesis.eden',
-      'EOS'
-    )
-    const data = await response
-
-    setCurrencyBalance(data[0] || '')
+    try {
+      const response = eosApi.getCurrencyBalance(
+        'eosio.token',
+        'genesis.eden',
```
Comment (leisterfrancisco) [inline webapp/src/hooks/customHooks/useIncomeReportState.js:59]: use environment variables

File: webapp/src/language/es.json [hunk 3]
```diff
@@ -40,11 +40,13 @@
     "textCurrencySelect": "Moneda",
     "textDelegateSelect": "Delegado",
     "titleTable": "Delegados ordenados por ingresos de mayor a menor",
-    "titleTable2": "Elecciones ordenadas de la mas reciente a la mas vieja",
+    "titleTable2": "Elecciones ordenadas de la mas vieja a la mas reciente",
```
Comment (leisterfrancisco) [inline webapp/src/language/es.json:43]: why if we use `antigua ` instead of `vieja`

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**